packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -276,6 +286,17 @@ This can be used only together with build type ${input('release')} for Android a
     this.env.log.ok(`Build completed: ${filename}`);
   }
 
+  async getDefaultArtifactType(platform: string): Promise<string> {
+    switch (platform) {
+      case 'ios':
+        return 'IPA';
+      case 'android':
+        return 'APK';
+      default:
+        throw new Error(`No default artifact type for platform '${platform}'`);
```

Comment:
Wasn't sure to do something simpler like `platform === 'ios' ? 'IPA' : 'APK'`, but did this incase we begin supporting other platforms e.g. `MacOS`, `Windows`, `Linux`.

Reply:
👍🏻